GOVERNANCE.md

```diff
@@ -31,7 +27,7 @@ changes to the project's governance processes.
 
 * Membership in the TC is granted by unanimous vote of current TC members.
 * TC members can step down at any time.
-* No more than 1/3 of TC members may belong to the same company.
+* No more than 1/3 (or 2, whichever is higher) of TC members may belong to the same company.
```

Comment:
proposing a substantive change here to better reflect the reality when we have so few maintainers. it's (IMO) unreasonable to expect that in a group of 4 maintainers, 2 of them could not belong to the same company. But in a group of, say, 9, we certainly wouldn't want 4 of them to be.